addons/point_of_sale/static/src/js/models.js

```diff
@@ -3105,7 +3105,7 @@ class Order extends PosModel {
                 var rounding_method = this.pos.cash_rounding[0].rounding_method;
                 var remaining = this.get_total_with_tax() - this.get_total_paid();
                 var sign = this.get_total_with_tax() > 0 ? 1.0 : -1.0;
-                if(this.get_total_with_tax() < 0 && remaining > 0 || this.get_total_with_tax() > 0 && remaining < 0) {
+                if(rounding_method !== "HALF-UP" && (this.get_total_with_tax() < 0 && remaining > 0 || this.get_total_with_tax() > 0 && remaining < 0)) {
                     rounding_method = rounding_method.endsWith("UP") ? "DOWN" : "UP";
```

Comment:
Since it cannot be "half-up" anymore, can you modify the ternary to: rounding_method === "up" ? "DOWN" : "UP";

Reply:
Thanks for your review.
I modified it.